# files for jaSteInte3

* braker.codingseq.gz - coding seqs from BRAKER3 predictions
* braker.aa.gz - proteins from BRAKER3 predictions
* braker.gtf.gz - GTF style annoations from BRAKER3
* jaSteInte3.emapper.decorated.gff.gz - GFF style annoations with EggNOG-mappe decoration from BRAKER3
* jaSteInte3.emapper.annotation.gz - Annotation output from EggNOG-mapper

# files hosted elsewhere
* [softmasked genome FASTA](https://asg_hubs.cog.sanger.ac.uk/jaSteInte3/jaSteInte3.fa.masked)
* [tarball of RepeatModeller output](https://asg_hubs.cog.sanger.ac.uk/jaSteInte3/jaSteInte3.tar.xz)
* [BAM file](https://asg_hubs.cog.sanger.ac.uk/jaSteInte3/VARUS_modified.bam) of VARUS sampled RNASeq from SRA (max 30 million spots)

# Statistics:

---
 * genes: 41611
 * average_gene_length: 6167
 * transcripts_per_gene: 1.1300857946216145
 * average_transcript_length: 1302
 * exons_per_transcript: 5.563095440626063
 * average_exon_length: 234


![Plot of BUSCO results](jaSteInte3_busco.jpeg)

# NOTE:

No RNA was obtained from jaSteInte3, the prediction of the genes were perfomed using the Protein database only
