# files for jaSidRadi1

* braker.codingseq.gz - coding seqs from BRAKER3 predictions
* braker.aa.gz - proteins from BRAKER3 predictions
* braker.gtf.gz - GTF style annoations from BRAKER3
* jaSidRadi1.emapper.decorated.gff.gz - GFF style annoations with EggNOG-mappe decoration from BRAKER3
* jaSidRadi1.emapper.annotation.gz - Annotation output from EggNOG-mapper

# files hosted elsewhere
* [softmasked genome FASTA](https://asg_hubs.cog.sanger.ac.uk/jaSidRadi1/jaSidRadi1.fa.masked)
* [tarball of RepeatModeller output](https://asg_hubs.cog.sanger.ac.uk/jaSidRadi1/jaSidRadi1.tar.xz)
* [BAM file](https://asg_hubs.cog.sanger.ac.uk/jaSidRadi1/VARUS_modified.bam) of VARUS sampled RNASeq from SRA (max 30 million spots)

# Statistics:

---
 * genes: 40258
 * average_gene_length: 5625
 * transcripts_per_gene: 1.1460082468080879
 * average_transcript_length: 1206
 * exons_per_transcript: 5.023452401595284
 * average_exon_length: 240


![Plot of BUSCO results](jaSidRadi1_busco.jpeg)

